app/controllers/rubocop_controller.rb

```diff
@@ -0,0 +1,11 @@
+class RubocopController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def webhook
+    RestClient.post(
+      'https://api.github.com/repos/Youssefares/python-error-helper-web/statuses/',
+      { 'state': 'success'}.to_json,
+      { content_type: :json, accept: "application/vnd.github.v3+json" }
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.